.github/ISSUE_TEMPLATE/bug_template.md

```diff
@@ -16,11 +16,17 @@ Steps to reproduce the behavior:
 3. Scroll down to '....'
 4. See error
 
+**Assets required**
+Provide sample assets that are needed to reproduce the issue.
+
 **Expected behavior**

```

Comment:
Can you also add **Actual behavior** while you're in here?

Reply:
Can you shorten the text to "Provide sample assets needed to reproduce the issue."

Or, perhaps rephrase to match the text under Screenshots - i.e. "If applicable, add asset files to aid in reproducing the issue."

Leaning towards the latter, but interested in your thoughts.

Reply:
@monroegm done

@forhalle I'd prefer your first one actually as the second one sounds like it is possible to reproduce the issue without them, but using them it helps seeing the issue - this might work for some issues but not for all. I'd like to go the safe route here and be as concrete as possible. If an asset is required to reproduce the bug, it should be required to attach it - as otherwise we end up with an issue that needs further communication as it cannot be reproduced.

How does the updated version look like? Shall I get any further modifications in?